ramp-database/rampdb/testing.py

```diff
@@ -0,0 +1,244 @@
+import logging
+import os
+import shutil
+import subprocess
+
+from git import Repo
+
+from ramputils import generate_ramp_config
+
+from .utils import setup_db
+from .utils import session_scope
+
+from .model import Model
+
+from .tools.database import add_extension
+from .tools.database import add_submission_file_type
+from .tools.database import add_submission_file_type_extension
+from .tools.event import add_event
+from .tools.event import add_problem
+from .tools.user import approve_user
+from .tools.user import create_user
+from .tools.team import sign_up_team
+from .tools.submission import submit_starting_kits
+
+logger = logging.getLogger('DATABASE')
+
+
+def create_test_db(config):
+    """Create an empty test database and the setup the files for RAMP.
+
+    Parameters
+    ----------
+    config : dict
+        Configuration file containing all ramp information.
```

Comment:
yes it is the top level one. Do you think that it would be better to pass a file instead?

An original idea of ramp-utils was to store some template of configs (website config, server config, event config, ...). The testing module from ramp-utils will return the path to the config. This is bit what I thought about passing the dict everywhere around.

Reply:
Oh just realize what you mean by your comment.